src/parser/Parser.java

```diff
@@ -69,12 +70,21 @@ public float getDailyWorkHours(String data) throws WrongFormatException, ParseEx
 					final Date date1 = format.parse(processedFile.getRows().get(i).getStartHour());
 					final Date date2 = format.parse(processedFile.getRows().get(i).getEndHour());
 					final float difference = TimeUnit.MILLISECONDS.toMinutes(date2.getTime() - date1.getTime());
-					oreLavorate = oreLavorate + difference;
+					minLavorati = minLavorati + difference;
 				}
 			}
 		} catch (final ParseException e) {
 			throw new WrongFormatException("Error: ",e);
 		}
+		
+		oreLavorate = (int) minLavorati / 60;
+		minLavorati = minLavorati % 60;
+		if (minLavorati != 10) {
+			oreLavorate = oreLavorate + minLavorati / 100;
+		} else {
```

Comment:
Con minLavorati = 70 le ore lavorate diventano 2 invece di 1.10 . Come mai se i minuti restanti sono 10 lo gestisci come caso a se?

Reply:
> 
> 
> Con minLavorati = 70 le ore lavorate diventano 2 invece di 1.10 . Come mai se i minuti restanti sono 10 lo gestisci come caso a se?

Perche' mi aveva tratto in inganno il minLavorati = 10, ora fatto il pull del codice giusto